app/views/accounts/_edit_form.html.erb

```diff
@@ -1,49 +0,0 @@
-<%= form_for(@account, class: 'form') do |f| %>
```

Comment:
Was this file dead code? Are we providing users the ability to tweak these endpoints elsewhere?